app/mailers/notifier.rb

```diff
@@ -15,6 +15,13 @@ def intro(user, mission: nil)
          subject: t("notifier.welcome", site: Settings.site_name))
   end
 
+  def sms_token_change_alert(mission)
+    @mission = mission
+    @site_name = Settings.site_name
+    mail(to: reply_to(mission), reply_to: reply_to(mission),
+         subject: t("notifier.sms_token_change.subject", mission_name: mission.name))
```

Comment:
Currently this is sent to all mission coordinators as specified in the `reply_to` method. Is "admin" a different type of user entirely or is this okay?

Is there guaranteed to be at least one coordinator, assuming this token is being regenerated by a user (in other words, can someone besides a coordinator regenerate it)?

Reply:
This makes sense but can we change the method name to e.g. `coordinator_emails`.

There is also a risk that an admin may not be a member of a mission that they are actively involved in because we don't make it obvious that they're not. It might be a good idea to add a kind of notification to the UI if this is the case... Let's ask this tomorrow at standup.